PosteAPI/urls.py

```diff
@@ -1,11 +1,11 @@
-from django.urls import include, path, re_path
+from django.urls import include, path
 
-from .views import UserAPI, UserDetail, UserLogin
+from .views import UsersView, UserDetail, LoginView, DataView
 
 urlpatterns = [
-    # GET to list users, POST to create a user
-    path("users/", UserAPI.as_view(), name="user-list"),
+    path("users/", UsersView.as_view(), name="user-list"),  # GET to list users, POST to create a user
     path("users/<int:pk>/", UserDetail.as_view(), name="user-detail"),
-    path("login/", UserLogin.as_view(), name="user-login"),
+    path("login/", LoginView.as_view(), name="user-login"),  # POST to login
+    path("data/", DataView.as_view(), name="user-folders"),  # Functions as data retrieval / update endpoint
```

Comment:
verifying comprehension: this endpoint would be used to update folder name, post name, folder permissions, etc.

Reply:
I think the verbiage I used was confusing, as this should not provide functionality to update backend data. 

The intent is simply to provide a 'bulk refresh' endpoint, either for initial population of data on login, or whenever a user requires a 'master refresh' of data (such as 'dragging down' from the top of the screen to refresh).  

Since this should only serve as a client update endpoint and not for updating backend data, I'll change the wording to indicate. Thank you!